ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml

```
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.17
+    tag: golang-1.19
```

```suggestion
    tag: golang-1.20
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44408/rehearse-44408-pull-ci-openshift-velero-plugin-for-csi-main-unit-test/1713989369158701056#1:build-log.txt%3A194